src/encoded/static/components/browse/columnExtensionMap.js

```diff
@@ -230,4 +230,31 @@ export const columnExtensionMap = _.extend({}, basicColumnExtensionMap, {
     'experiments_in_set.biosample.biosource_summary' : {
         'widthMap' : { 'lg' : 260, 'md' : 230, 'sm' : 180 }
     },
+    'date_created' : {
```

Comment:
We can remove `date_created` from here since this extends basicColumnExtensionMap from SPC which has it already.

Reply:
@alexkb0009  date_created of basicColumnExtensionMap is right-aligned but here we override it as center-aligned. if we remove it, Created column and Modifided columns' styles are not matching.

Reply:
Good point, sorry didn't pay much attention re: the 'centering' part in prev comments --

I think the most important thing above all here is to have all date columns render in the same exact style. Whether that means is text-center or text-right, I guess is somewhat subjective and not that important.

But if we're gonna change to have these 2 date columns be `text-center` instead of `text-right`, to stay consistent, we should change this for _all_ columns, including `google_analytics.for_date` and `date_published ` within this file as well as for the date fields in SPC, and the ones in CGAP.

(I'd argue that keeping all dates be `text-right` would be preferable, both because less code changes needed from current state, and (subjective/arguably-) aligning the year might help to see differences in date more easily when many rows are shown)